cg/meta/upload/mip/mip_rna.py

```diff
@@ -34,14 +34,16 @@ def upload(self, ctx: click.Context, case: Family, restart: bool) -> None:
 
         # Scout specific upload
         if DataDelivery.SCOUT in case.data_delivery:
-            ctx.invoke(upload_rna_to_scout, case_id=case.internal_id)
+            result: int = ctx.invoke(upload_rna_to_scout, case_id=case.internal_id)
+            if result == 0:
+                LOG.info(
+                    f"Upload of case {case.internal_id} was successful. Setting uploaded at to {dt.datetime.now()}"
+                )
+                self.update_uploaded_at(analysis_obj)
```

Comment:
Refactor to just analysis
```suggestion
                self.update_uploaded_at(analysis)
```